src/main/java/com/opentable/kafka/util/OffsetMetricsBuilder.java

```diff
@@ -0,0 +1,82 @@
+package com.opentable.kafka.util;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.codahale.metrics.ExponentiallyDecayingReservoir;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Reservoir;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Supplier;
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * Get one of these by calling {@link OffsetMetrics#builder}; provide at least one topic.
+ */
+public class OffsetMetricsBuilder {
+    private final Set<String> topics = new HashSet<>();
+
+    private final String metricPrefix;
+    private final MetricRegistry metricRegistry;
+    private final String groupId;
+    private final String brokerList;
+
+    private Duration pollPeriod = Duration.ofSeconds(10);
+    private Supplier<Reservoir> reservoirSupplier = ExponentiallyDecayingReservoir::new;
+
+    OffsetMetricsBuilder(
+            final String metricPrefix,
+            final MetricRegistry metricRegistry,
+            final String groupId,
+            final String brokerList
+    ) {
+        this.metricPrefix = metricPrefix;
+        this.metricRegistry = metricRegistry;
+        this.groupId = groupId;
+        this.brokerList = brokerList;
+    }
+
+    public OffsetMetricsBuilder withTopics(final Collection<String> topics) {
```

Comment:
might be worth specifying that this setter, unlike the others, is additive rather than replacing the existing value.

Reply:
Oh, great point.  You could totally mistake it otherwise.  I'll rename all of the "topics" methods and update agassoc and chat accordingly.

Reply:
Implemented in 69acb88a89e6b7b80957d729e1b40270c24e4ae0.  Updated https://github.com/opentable/service-agassoc/pull/22 and https://github.com/opentable/service-chat/pull/95 as well.